docs/release-process/CHECKLIST.md

```diff
@@ -0,0 +1,84 @@
+## Check list
+
+This section shows what things you need to do before creating a release. This is a manual process as CI does not test everything.
+
+The following example assumes we’re going from version 0.1.0 (`v0.1.0`) to
+0.2.0 (`v0.2.0`).
+
+- Checkout to old release tag
+  - e.g. `git checkout v0.1.0`
+
+- Build `lokoctl` binary from the last release
+  - e.g. `make build`
+  - Copy `lokoctl` binary to your assets directory.
+
+- Deploy `lokomotive` with old release
+  - e.g. `./lokoctl cluster apply`
+  - Deploy a single cluster(controller node) first.
+
+- Configure env variable `$KUBECONFIG` as per your assets directory location.
+
+- Make sure all nodes healthy and all Pod are running.
+  - e.g. `kubectl get po -A && kubectl get no`
+
+- Delete the old `lokoctl` binary.
+
+- Checkout to latest `master` branch. Make sure you have the latest code fetched and rebased.
+  - e.g. `git checkout master && git fetch origin && git rebase origin/master`
+  - Build lokoctl binary from latest master `make build`
+
+- Copy `lokoctl` binary to your assets directory and re-apply the cluster
+  - e.g. `./lokoctl cluster apply`
+
+- Make sure all nodes healthy and all Pod are running.
+  - e.g. `kubectl get po -A && kubectl get no`
+
+### Components test
+
+This sections checks if components work as desired. 
+
+- Check all certificates are valid
+  - e.g. `kubectl get certificates -A`
+  - Certificates for all your components are valid.
+
+- Check external IP is assigned to contour service. This will verify that MetalLB is assigning IP to service of type `LoadBalancer`.
+  - `kubectl get svc -n projectcontour`
+
+- Check routes are added to AWS for your components. If you have used route53 DNS provider, you can check them [here](https://console.aws.amazon.com/route53/v2/home#Dashboard). Make sure to check the correct hosted zone.
+
+- Check Gangway Ingress Host URL that you have configured works fine.
+
+- Check httpbin Ingress Host URL that you have configured works fine.
+
+- Do some **blackbox testing** by sending HTTP requests through MetalLB + Contour + cert-manager.
+
+- Check metrics for your cluster by going to Prometheus Ingress Host URL.
```

Comment:
- We can point the user to the prometheus doc and ask them to verify all the different scenarios in there if they work.
- Verify if the grafana dashboards have data.
- Verify if the prometheus targets are loaded correctly.
- Verify if the alerts are loaded correctly.

Reply:
https://github.com/kinvolk/lokomotive/blob/master/docs/how-to-guides/monitoring-with-prometheus-operator.md

Reply:
Most of the points here would be already covered by e2e test suite, right? Prometheus data and targets.

Perhaps we should add tests for alerts and grafana data sources then.